### PR TITLE
Adds validations for models

### PR DESCRIPTION
**Pull request by leezhiqin96**
# Description

Added validations for Job, Bid and User models.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Screenshot A
- [ ] Screenshot B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added proof(eg. screenshots) that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by deepikajoshi1 — COMMENTED** [review]
ok.. looks good..

File: app/models/job.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
 class Job < ApplicationRecord
   belongs_to :user, class_name: "User", foreign_key: :employer_id
   has_many :users, through: :bids
+  has_many :bids
+
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :budget, presence: true
+  validates :start_date, presence: true
+  validates :end_date, presence: true
```
Comment (jwltay) [inline app/models/job.rb:10]: maybe validate that end date comes after start date?

Comment (leezhiqin96) [reply app/models/job.rb:10]: Ok sounds good! I will look into it

Comment (jwltay) [reply app/models/job.rb:10]: seems good, thanks! will test together with my controllers

**Review by jwltay — APPROVED** [review]

File: app/models/job.rb (hunk 1 above)
Comment (zonghangoh) [reply app/models/job.rb:10]: love this. good job.

**Pull request merged**